docs/learn/bitesize/bitesize-usecases.md

```diff
@@ -17,6 +17,17 @@ Each scene includes a tutorial to help you locate the scripts and GameObjects it
 
 The tutorials that open with each scene use the [Tutorial Framework package](https://docs.unity3d.com/Packages/com.unity.learn.iet-framework@4.0/manual/index.html). You can open each tutorial at any time from the **Tutorials** menu.
 
+## The Anticipation scene
+
+The Anticipation scene demonstrates the Client Anticipation feature of Netcode for GameObjects in the following use cases: 
+- **AnticipatedNetworkVariable:**
+  - Altering NetworkVariables responsively by anticipating server actions based on player interaction.
```

Comment:
```suggestion
  - Anticipate server actions based on player interaction to change NetworkVariables responsively. 
```

Reply:
What do the NetworkVariables respond to?

Reply:
I'm not sure of what you mean with your question. What we want to say is that "we locally alter the value of NetworkVariables, without waiting for a confirmation of the server".

All this stuff is basically "client-side prediction"